api/type.rb

```diff
@@ -90,10 +90,20 @@ def type
       self.class.name.split('::').last
     end
 
+    # This is only used in puppet and chef, and it is the name of the Ruby type
```

Comment:
We should probably move these outside of the api/ folder and into a puppet/chef specific provider file at some point.

Reply:
Definitely!  But for now, some documentation is better than no documentation.